tests/validator/test_validation_graph.py

```diff
@@ -439,23 +439,27 @@ class DummyExecutionEngine:
     execution_engine = cast(ExecutionEngine, DummyExecutionEngine)
 
     # ValidationGraph is a complex object that requires len > 3 to not trigger tqdm
-    with mock.patch(
-        "great_expectations.validator.validation_graph.ValidationGraph._parse",
-        return_value=(
-            {},
-            {},
+    with (
```

Comment:
is this formatting change intended to be part of this PR?

Reply:
Yes it's a syntax/formatting change made possible by bumping our min version of python to 3.9 and updating the `ruff` "python version target".